.github/workflows/changesets.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches:
       - main
 
+permissions:
+  # content:write and pull-request:write permissions needed to create changeset PRs
+  contents: write
+  pull-requests: write
```

Comment:
This shouldn't be needed. We use `GH_ACCESS_TOKEN` to create the PR, not `GITHUB_TOKEN`

Reply:
oh! right! good call! 👍